.github/workflows/pr-gpu.yaml

```
@@ -16,7 +16,7 @@ jobs:
   pytest-gpu:
     name: ${{ matrix.name }}
     if: github.repository_owner == 'databricks'
-    runs-on: ubuntu-latest # todo: switch to linux-ubuntu-latest later
+    runs-on: ubuntu-latest  # todo: switch to linux-ubuntu-latest later
```

```suggestion
    runs-on: ubuntu-latest  # TODO: switch to linux-ubuntu-latest later
```
nit consistent formatting

---

Can you file a JIRA for this

---

Done.